appservice/src/createAppService/setLocationsTask.ts

```diff
@@ -32,7 +32,15 @@ export async function getWebLocations(context: IAppServiceWizardContext): Promis
         options.sku = <SkuName>context.newPlanSku.tier.replace(/\s/g, '');
     }
 
-    const client: WebSiteManagementClient = await createWebSiteClient(context);
-    const locations = await uiUtils.listAllIterator(client.listGeoRegions(options));
-    return locations.map(l => nonNullProp(l, 'name'));
+    const genericClient = await createGenericClient(context, context);
```

Comment:
Add the github issue as a comment so we know to revisit this when it's resolved.